src/hooks/useConnectionStatus.tsx

```diff
@@ -30,6 +33,13 @@ export const ConnectionStatusProvider: React.FC = ({ children }) => {
     }
   }, [])
```

Comment:
I think, we can get rid of this warning easily by adding `checkConnection` to the dependency array and wrapping `checkConnection` in useCallback